ext/blocks/functions.js

```diff
@@ -156,12 +160,14 @@ Blockly.Blocks.function_call = {
 
         this.params = {};
     },
-    getFunctionCall () {
+    getFunctionCall() {
         return this.getFieldValue('NAME');
     },
-    updateShape () {
+    updateShape() {
         let connectionHistory = {},
-            input, funcDef, updateConnections;
+            input,
```

Comment:
This isn't the usual pattern we use other places. Would it make more sense to declare funcDef on line 173 and input inside the foreach on line 182, as we don't reassign them? More in keeping with the style of the other codebases